config/sync/field.storage.config_pages.su_site_algolia_key.yml

```diff
@@ -1,15 +1,11 @@
-uuid: 78362d9b-2be0-4817-a1f0-2491186b16d7
+uuid: 8b96ad74-d051-4640-9e93-63b9d08474eb
```

Comment:
`uuid` should never change.

Reply:
Oh, I missed this. Link to the User Guide is a good idea.
Also, trying to future out why that UUID would have changed... I

Reply:
For some reason this PR hates me. Should have a new version later... but I might start from scratch at this point.